findbugs-test-util/src/test/java/com/h3xstream/findbugs/test/service/ClassFileLocator.java

```diff
@@ -63,12 +63,17 @@ public String getJarFilePath(String path) {
 
     private String getFilenameFromUrl(URL url) {
         String filename;
-		try {
-			filename = url.toURI().getPath();
-		} catch (final URISyntaxException e) {
-			fail("Failed to get file path = " + url, e);
-			return null;
-		}
+        try {
+            if (url.getPath().contains(".jar!")) {
+                String jarRegex = "findsecbugs-test-kotlin-[\\d\\.]{5,}-tests\\.jar!";
```

Comment:
Best effort to resolve files from the test-kotlin module.  Existing URI resolution resolved to null